nextflow.config

```diff
@@ -63,9 +67,8 @@ params {
     pon = null // No default PON (Panel of Normals) file for GATK Mutect2 / Sentieon TNscope
     pon_tbi = null // No default PON index for GATK Mutect2 / Sentieon TNscope
     ignore_soft_clipped_bases = false // no --dont-use-soft-clipped-bases for GATK Mutect2
-    umi = false // no umi
-    read_structure1 = null // no umi
-    read_structure2 = null // no umi
+    wgs = false // Set to true, if data is whole genome sequencing data. Used to use correct models in various variant callers
```

Comment:
Can't we assess that with the presence/absence of target_bed?

Reply:
Shouldn't default be `wgs = true`? or is `wes` the default?
Should we ask our community what they would prefer?

Reply:
sure either is fine for me. Lets make a poll/thread